### DISCUSSION TITLE
How to access daily backups?

### DISCUSSION THREAD

**Discussion by cliveportman** [supabase/supabase#10490] [Questions] [answered]
I'm on the Pro tier, and need to look at yesterday's database. Where are these backups?

**Comment by GaryAustin1** [comment, marked as answer]
Are they not showing up in the dashboard under database then backups?

If not you may want to contact support.

**Reply by cliveportman** [reply]
They are. Seems silly I didn't find it but I kind of expect to see this stuff documented somewhere. Googling 'supabase restore database' returns nothing.
